Route/RoutePageGenerator.php

```diff
@@ -99,9 +99,20 @@ public function update(SiteInterface $site, OutputInterface $output = null)
         }
 
         // Iterate over declared routes from the routing mechanism
+        $routesDedupInfos = array();
+        // We first de-duplicate routes by path; the first declaration win !
         foreach ($this->router->getRouteCollection()->all() as $name => $route) {
-            $name = trim($name);
+            $routePath = $route->getPath();
+            if (array_key_exists($routePath, $routesDedupInfos)) {
+                $this->writeln($output, sprintf('  <info>IGNORE</info> route %s is ignored because it uses the same path as %s (path=%s)',
+                    $name, $routesDedupInfos[$routePath][0], $routePath));
```

Comment:
Looks like there is a PR : https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1187